<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectForm</class>
 <widget class="QWidget" name="NewProjectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color: #84f1ba;
color:  #394040;
}
QLineEdit
{
background-color: #ffffff;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="CustomListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>384</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget
{
background-color: #dcf7f7;
color:#394040;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="movement">
        <enum>QListView::Free</enum>
       </property>
       <property name="flow">
        <enum>QListView::TopToBottom</enum>
       </property>
       <property name="layoutMode">
        <enum>QListView::SinglePass</enum>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>384</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget
{
background-color: #84f1ba;
color:  #394040;
}</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_5">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>591</width>
           <height>241</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QTextBrowser
{
	border: 1px;
}
</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:12pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Master of new project creation&lt;/span&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;&lt;br /&gt;Follow this master you could create a new project with different diagrams.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="projectNameLine">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>471</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ffffff;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushBrowsePathButton">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>80</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="projectPathLine">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>80</y>
           <width>381</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ffffff;</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Choose template</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>191</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ffffff;</string>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Activity diagram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Class diagram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Communication diagram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Deployment diagram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Precendence diagram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sequence diagram</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Choose language</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ffffff;</string>
         </property>
         <item>
          <property name="text">
           <string>C++</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Java</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>C#</string>
          </property>
         </item>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>441</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #ffffff;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="BrowseCodeButton">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>50</y>
           <width>81</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>File to be added in</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="labelTemplate">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Source code path:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="labelFileName">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>200</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelFilePath">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Path:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="labelSourceCode">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Template:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PrevPageButton">
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NextPageButton">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">CustomStackedWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">CustomListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
